isb_web/main.py

```diff
@@ -77,6 +77,7 @@
 manage.dao = dao
 metrics.dao = dao
 vocabulary.dao = dao
+export.dao = dao
```

Comment:
these global dao vars always seem a bit odd, but it seems to be the way it's done in FastAPI...

Reply:
There could be a better way to do it using the dependency injection features of FastAPI, but I haven't really spent time investigating.